.eslintrc.json

```diff
@@ -8,6 +8,7 @@
       "project": "./tsconfig.json"
     },
     "rules": {
+      "no-console": "off",
```

Comment:
Logging to console is visible in github actions UI so I find console.log useful and therefore allowed.